build.gradle

```diff
@@ -254,9 +257,9 @@ processResources {
 }
 
 processTestResources {
-    //Don't waste time packaging our test data into the test jar
-    exclude "**/large/**"
-    exclude "**/org/broadinstitute/hellbender/**"
+    //Don't waste time packaging unnecessary test data into the test jar:
```

Comment:
I think we may still want the excludes, I'm not certain how it works but I thought that it would include src/test/resources/ by default.

Reply:
Fixed in latest commit.